src/modules/70-pipeline/components/PipelineSteps/Steps/Approval/HarnessApprovalStepMode.tsx

```diff
@@ -324,10 +324,16 @@ function HarnessApprovalStepMode(
               .test({
                 test(value: string) {
                   if (getMultiTypeFromValue(value) === MultiTypeInputType.FIXED && !value.match(regexPositiveNumbers)) {
-                    return this.createError({
-                      message: getString('pipeline.approvalStep.validation.minimumCountOne')
-                    })
+                    if (value != '0' && value.indexOf('.') !== -1)
```

Comment:
Any reason for using `!=` instead of `!==`?
```suggestion
                    if (value !== '0' && value.includes('.'))
```

Reply:
No , I missed strict inequality here.